util/MultiplayerCommon.h

```diff
@@ -51,6 +51,9 @@ FO_ENUM(
     ((GALAXY_SETUP_MEDIUM))
     ((GALAXY_SETUP_HIGH))
     ((GALAXY_SETUP_RANDOM))
+    ((GALAXY_SETUP_MICROSCOPIC))    // backwards-compatibility -> new enum values to the back
```

Comment:
remove the comment 3x

Reply:
this fragment is outdated, sorry, already changed that to a new enum with fewer comments in a following commit.